src/subdomains/generic/kyc/services/kyc-admin.service.ts

```diff
@@ -55,7 +55,7 @@ export class KycAdminService {
   async triggerWebhook(dto: KycWebhookTriggerDto): Promise<void> {
     const kycStep = await this.kycStepRepo.findOne({
       where: [{ id: dto.kycStepId }, { userData: { id: dto.userDataId } }],
-      relations: { userData: true },
+      relations: { userData: { users: { wallet: true } } },
       order: { updated: 'DESC' },
```

Comment:
I'm not sure if we should load all users with wallets everywhere. It will probably slow down everything. Maybe we can just load the users when we store the webhook?